website/source/docs/guides/index.html.md

```diff
@@ -28,7 +28,7 @@ Tracks include:
 - [Getting Started with Kubernetes](https://learn.hashicorp.com/consul/?track=getting-started-k8s#getting-started-k8s)
 - [Day 1: Deploying Your First Datacenter](https://learn.hashicorp.com/consul/?track=datacenter-deploy#datacenter-deploy)
 - [Day 1: Security and Network Operations](https://learn.hashicorp.com/consul/?track=security-networking#security-networking)
-- [Day 1: Kubernetes Operations](https://learn.hashicorp.com/consul/?track=kubernetes#kubernetes)
+- [Day 1: Kubernetes Operations](https://learn.hashicorp.com/consul/?track=day-1-operations#day-1-operations)
```

Comment:
When I go to this I can't actually see the intended content as it is above the anchor. Is there a way to get around that? Could we link to a specific guide?

Reply:
We could link to the first guide in each track. @kaitlincarter-hc do you have thoughts?

Reply:
Talked this over with Kait: Not all of the tracks have index pages and the top level guides page is easy to navigate, so we decided to replace the links to specific tracks with a CTA pointing to the learn platform instead. We can add track links back once the anchor links get fixed

Reply:
👍

Reply:
👍